cohort_4/playbook/ACTIVITIES.md

```diff
@@ -880,11 +880,42 @@ shouted out after the demo.
 # Freeform Projects
 
 This assumes the hackers are already in groups and have decided what to work on.
-If they have not yet, refer to [Brainstorming](#brainstorming).
+
+If they have not yet, refer to [Brainstorming](#project-formation).
+
+## Scheduling
+
+- make sure to collect links 10 minutes before demos at around 3:50.
+- make sure you remind people of what time the demos is. It is very easy to forget to mention the time.
+
+## Role Of The Organizer
+
+Walking the rope of how to be effecive during this time period can be hard. On one hand, you want to actively help hackers. At the same time, you don't want to hover and so you stay back and let the hackers do their thing and work indepenently.
```

Comment:
Any updates on this?

On Fri, Aug 7, 2015 at 6:57 PM, Max Wofford <notifications@github.com>
wrote:

> In cohort_4/playbook/ACTIVITIES.md
> <https://github.com/hackedu/hack-camp/pull/394#discussion_r36573667>:
>
> > @@ -880,11 +880,42 @@ shouted out after the demo.
> >  # Freeform Projects
> >
> >  This assumes the hackers are already in groups and have decided what to work on.
> > -If they have not yet, refer to [Brainstorming](#brainstorming).
> > +
> > +If they have not yet, refer to [Brainstorming](#project-formation).
> > +
> > +## Scheduling
> > +
> > +- make sure to collect links 10 minutes before demos at around 3:50.
> > +- make sure you remind people of what time the demos is. It is very easy to forget to mention the time.
> > +
> > +## Role Of The Organizer
> > +
> > +Walking the rope of how to be effecive during this time period can be hard. On one hand, you want to actively help hackers. At the same time, you don't want to hover and so you stay back and let the hackers do their thing and work indepenently.
>
> This should be wrapped at 80 chars.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/hack-camp/pull/394/files#r36573667>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us